Sources/MountebankSwift/Models/Config/Logs.Log+Codable.swift

```diff
@@ -13,7 +13,7 @@ extension Logs.Log {
         try container.encode(level, forKey: .level)
         try container.encode(message, forKey: .message)
         if let timestamp {
-            try container.encodeIfPresent(DateFormatter.shared.formatFromDate(timestamp), forKey: .timestamp)
+            try container.encodeIfPresent(DateFormatter().formatFromDate(timestamp), forKey: .timestamp)
```

Comment:
This is expensive. 

Can we perhaps do 

```
        if #available(macOS 12.0, iOS 15.0, tvOS 15.0, watchOS 8.0, *) {
            try container.encodeIfPresent(timestamp.formatted(), forKey: .timestamp)
        } else {
            try container.encodeIfPresent(DateFormatter().formatFromDate(timestamp), forKey: .timestamp)
        }
```

Or cache the formatter some other way?